titus-supplementary-component/task-relocation/src/main/java/com/netflix/titus/supplementary/relocation/descheduler/DefaultDeschedulerService.java

```diff
@@ -77,11 +77,8 @@ public DefaultDeschedulerService(ReadOnlyJobOperations jobOperations,
 
     @Override
     public List<DeschedulingResult> deschedule(Map<String, TaskRelocationPlan> plannedAheadTaskRelocationPlans) {
-        List<Pair<Job, List<Task>>> allJobsAndTasks = jobOperations.getJobsAndTasks();
```

Comment:
The reason why we should use this method is to have a consistent snapshot of the data. If you ask for the jobs first, and next for tasks, it is possible that you get tasks for which there are no jobs in the`jobs` collection.
I think we should make a fix but handling properly the result which is `List<Pair<Job, Map<String,Task>>>`.